connector/connector.py

```diff
@@ -443,10 +443,10 @@ def unwrap_binding(self, binding_id, browse=False):
         else:
             binding = self.model.browse(binding_id)
 
-        openerp_record = getattr(binding, self._openerp_field)
+        record = getattr(binding, self._openerp_field)
```

Comment:
[This is a security issue](https://github.com/odoo/odoo/issues/11087). Does `binding[self._openerp_field]` work?